src/mitol/payment_gateway/api.py

```diff
@@ -135,6 +149,25 @@ def start_payment(
 
         return GatewayCls.prepare_checkout(order, receipt_url, cancel_url, **kwargs)
 
+    @classmethod
+    def validate_processor_response(cls, payment_type, request):
+        """
+        Kicks off validation of a processor response.
+
+        Args:
+            payment_type:   String; gateway class to use
+            request:        HttpRequest object or equivalent
+
+        Returns:
+            True or False (this is implementation-specific)
+        """
+        if payment_type not in cls._GATEWAYS:
+            return TypeError(f"{payment_type} not defined")
+
+        GatewayCls = cls._GATEWAYS[payment_type]()
```

Comment:
It's probably worth pulling this logic out into a separate method since it's being used in a few places now.